code/style.css

```diff
@@ -1,4 +1,190 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: white;
+  color: black;
 }
+
+body {
+  font-family: 'IBM Plex Sans KR', sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+}
+
+a {
+  font-family: 'Poppins', sans-serif;
+  text-decoration: none;
+  color: rgb(135, 133, 133);
+  font-weight: bold;
+}
+
+a:hover {
+  color: black;
+}
+
+header img {
+  height: 40px;
+  font-size: 16px;
+}
+
+img {
+  max-width: 100%; 
+  height: auto
+}
+
+header {
+  display: flex;
+  padding: 20px;
+}
+
+.logo-name {
+  display: none;
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  margin-left: 40px;
+  align-items: center;
+}
+
+.navigation {
+  display: none;
+}
+
+.big-news {
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  padding: 15px;
+  margin-bottom: 25px;
+  background-color: lightgray;
+  border-radius: 0.2cm;
+}
+
+.big-news-image {
+  overflow: hidden;
+  border-radius: 0.2cm;
+}
+
+.big-news-text {
+  overflow: hidden;
+}
+
+button {
+  background-color:grey;
+  border: none;
+  color: black;
+  padding: 5px;
+  border-radius: 0.2cm;
+}
+
+.small-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 25px;
+}
+
+.small-news-article {
+  display: flex;
+  height: 200px;
```

Comment:
The text is not visible in mobile view. I suggest using flex-direction: column and increasing the height.